src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/settings/EmailAddressAdder.tsx

```diff
@@ -105,14 +118,31 @@ const EmailAddressAddForm = () => {
         <label htmlFor="newEmailAddress">
           {l10n.getString("add-email-address-input-label")}
         </label>
-        <input type="email" name="newEmailAddress" id="newEmailAddress" />
-        <Button type="submit" variant="primary">
-          {l10n.getString("add-email-send-verification-button")}
+        <input
+          type="email"
+          name="newEmailAddress"
+          id="newEmailAddress"
+          onChange={handleInputChange}
+        />
+        <Button
+          type="submit"
+          variant="primary"
+          className={styles.btn}
+          disabled={!isEmailValid()}
+          onPress={() => {
+            setHasPressedButton(true);
+          }}
+        >
+          {hasPressedButton ? (
+            <Loader />
```

Comment:
We could also consider moving this into `<Button>` as an `isLoading` prop or something, since I can imagine us using this pattern elsewhere?

(In fact, the `<Button>` element could also add that aria-live message I mentioned above next to the button?)